canvas_sdk/commands/commands/assess.py

```diff
@@ -31,6 +32,14 @@ def values(self) -> dict:
         }
 
 
+class AssessCommandNoInitValidation:
+    """Assess Command without validation on initialization."""
+
+    def __new__(cls, **kwargs: dict) -> AssessCommand:
+        """Returns an initialized Assess Command without any validation."""
+        return AssessCommand.model_construct(**kwargs)
+
+
```

Comment:
a little bit of a hack to avoid init validation. for each command, i created a separate class where it returns the `model_construct` of the command class. then in the `canvas_sdk/commands/__init__.py` file, we import each of these `NoInitValidation` classes with the name of the real class. if someone ever wanted to use the real class that has init validation, they can import directly, like so: `canvas_sdk/commands/commands/assess.py`